frontend/routes/notifications/notification.item.tsx

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Each child component (if it is not shared) should be placed in `components` directory (in this case: `routes/notifications/components`).

Reply:
Also the child component should have each separate style file?